liquibase/pom.xml

```diff
@@ -19,13 +19,42 @@
 	<groupId>org.openmrs.liquibase</groupId>
 	<artifactId>openmrs-liquibase</artifactId>
 	<name>openmrs-liquibase</name>
+	<packaging>jar</packaging>
 	<description>Generate  running liquibase change sets from the console</description>
 
 	<dependencies>
+		<dependency>
+			<groupId>dom4j</groupId>
+			<artifactId>dom4j</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>jaxen</groupId>
+			<artifactId>jaxen</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
```

Comment:
the PR did not run tests on Travis. Is that because this module did not have tests or do we need to set something up?

Could you please use JUnit 5 instead of 4. We just migrated the entire openmrs-core. JUnit 4 is still around here for backwards compatibility for module devs. In core new tests should now be written in Junit 5. We still need to update the testing docs.

I think it should be enough If you depend on jupiter-api and engine. Then in IntelliJ you can easily convert your tests with the Refactor Migrate or individually. What we learned so far is docunented at https://wiki.openmrs.org/display/docs/How+to+migrate+to+JUnit+5

Reply:
Sure thing, will do.

Reply:
Done.